recipes/omamer/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "omamer" %}
```

Comment:
Please separate each recipe as a PR

Reply:
ok, but then I guess I will need to add one after each other so that the dependencies can be resolved?

Reply:
(omamer depends on PySAIS and rmath4)